todos-app/backend/task_service/spec/routing/labels_routing_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe LabelsController, type: :routing do
+  describe 'routing' do
+    it 'routes to #index' do
+      expect(get: '/labels').to route_to('labels#index')
+    end
+
+    it 'routes to #show' do
+      expect(get: '/labels/1').to route_to('labels#show', id: '1')
+    end
+
+    it 'routes to #create' do
+      expect(post: '/labels').to route_to('labels#create')
+    end
+
+    it 'routes to #update via PUT' do
+      expect(put: '/labels/1').to route_to('labels#update', id: '1')
+    end
+
+    it 'routes to #update via PATCH' do
+      expect(patch: '/labels/1').to route_to('labels#update', id: '1')
+    end
+
+    it 'routes to #destroy' do
+      expect(delete: '/labels/1').to route_to('labels#destroy', id: '1')
+    end
```

Comment:
It's my first time seeing a routing_spec. Where do you learn about this?

https://relishapp.com/rspec/rspec-rails/docs/routing-specs
> Simple apps with nothing but standard RESTful routes won't get much value from
routing specs, but they can provide significant value when used to specify
customized routes, like vanity links, slugs, etc.

Reply:
I see!
I thought you added it yourself. If it was auto generated, then just leave it there.

> I guess it's better than not having one?

Yeah, that's right.